test/data/enrichment/test_service.py

```diff
@@ -104,34 +103,33 @@ def test_enrichment_query_by_points_two_variables(self):
         tablename = 'test_table'
         geometry_column = 'the_geom'
         query_function = _prepare_sql_by_points
-        variables = pd.DataFrame([['project.dataset.category_table1_comp_geog_geogyear_yearly_datayear.var1'],
-                                  ['project.dataset.category_table2_comp_geog_geogyear_yearly_datayear.var1']],
-                                 columns=['id'])
+        variables = ['carto-do.ags.demographics_crimerisk_usa_blockgroup_2015_yearly_2018.CRMCYBURG',
+                     'carto-do.mastercard.financial_mrli_usa_blockgroup_2019_monthly_2019.ticket_size_score']
         filters = {'a': 'b'}
         kwargs = {'data_geom_column': geometry_column, 'variables': variables, 'filters': filters}
 
         queries = _enrichment_queries(user_dataset, tablename, query_function, **kwargs)
 
         expected_queries = ['''SELECT data_table.enrichment_id,
-            var1,
-            ST_Area(enrichment_geo_table.geom) AS var1_area,
-            NULL AS var1_population
+            CRMCYBURG,
+            ST_Area(enrichment_geo_table.geom) AS CRMCYBURG_area,
+            NULL AS CRMCYBURG_population
         FROM `carto-do-customers.{user_dataset}\
-            .dataset_category_table1_comp_geog_geogyear_yearly_datayear` enrichment_table
+            .ags_demographics_crimerisk_usa_blockgroup_2015_yearly_2018` enrichment_table
         JOIN `carto-do-customers.{user_dataset}\
-            .dataset_geography_comp_geog_geogyear` enrichment_geo_table
+            .ags_geography_usa_blockgroup_2015` enrichment_geo_table
         ON enrichment_table.geoid = enrichment_geo_table.geoid
         JOIN `carto-do-customers.{user_dataset}.{tablename}` data_table
         ON ST_Within(data_table.{geometry_column}, enrichment_geo_table.geom)
         WHERE a='b';''', '''
         SELECT data_table.enrichment_id,
-            var1,
-            ST_Area(enrichment_geo_table.geom) AS var1_area,
-            NULL AS var1_population
+            ticket_size_score,
+            ST_Area(enrichment_geo_table.geom) AS ticket_size_score_area,
+            NULL AS ticket_size_score_population
         FROM `carto-do-customers.{user_dataset}\
-            .dataset_category_table2_comp_geog_geogyear_yearly_datayear` enrichment_table
+            .mastercard_financial_mrli_usa_blockgroup_2019_monthly_2019` enrichment_table
```

Comment:
Let's avoid referencing premium datasets in our source code. Just in case we face some ©️ problems 👼

Reply:
We need any premium dataset to test because of the use case for a "premium dataset" and a public dataset is different. The first one is read from `carto-do-customers` project and the second one from `carto-do-public-data` project.